deployment/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  cordra:
+    build:
+      context: cordra
+      dockerfile: docker/Dockerfile
+    env_file:
```

Comment:
If you just put a file called `.env` at the position of the DockerFile you won't need to specify it here. https://docs.docker.com/compose/environment-variables/set-environment-variables/

Reply:
I'll have to include that into the image then since we are building it locally. Will do that separately from this PR